worker/src/Settings.cpp

```diff
@@ -10,9 +10,13 @@
 #include <cerrno>
 #include <iterator> // std::ostream_iterator
 #include <sstream>  // std::ostringstream
+#ifdef _MSC_VER
+#include "opt/getopt.h"
```

Comment:
Once in `worker/deps/getopt/` it should be `#include <getopt.h>`.

Reply:
already changed.